modules/globebrowsing/globebrowsingmodule.cpp

```diff
@@ -158,6 +159,16 @@ namespace {
 
         return result;
     }
+
+    struct [[codegen::Dictionary(GlobeBrowsingModule)]] Parameters {
+        std::optional<bool> cacheEnabled [[codegen::key("WMSCacheEnabled")]];
```

Comment:
Missing documentation? Why not use the `PropertyInfo`s specified above?

Reply:
Done;  this was one of the first classes before codegen::description or codegen::verbatim existed